src/manager.js

```diff
@@ -430,30 +500,29 @@ AdManager.prototype.loadAds = function(element, updateCorrelator) {
  * @param {domElement} DOM element containing the DFP ad slot
  * @returns undefined
 */
-AdManager.prototype.refreshSlot = function(domElement) {
+AdManager.prototype.refreshSlot = function (domElement) {
+  var that = this;
   if (this.options.amazonEnabled && this.amznads) {
     params = {
-      id: this.amazonId,
-      callback: this.amazonAdRefresh.bind(this, domElement),
-      timeout: 500
-    };
-    this.amazonAdRefreshThrottled(params);
```

Comment:
@spra85 Thanks for catching this. Looks like it got lost while merging/rebasing. Re-added it.